src/saving/SaveManagerGUI.cs

```diff
@@ -166,17 +172,39 @@ private void DeleteSelectedButtonPressed()
         deleteSelectedConfirmDialog.PopupCenteredMinsize();
     }
 
+    private void DeleteOldButtonPressed()
+    {
+        deleteOldConfirmDialog.DialogText =
+            "Deleting all old Auto and Quick saves cannot be undone, " +
+            "are you sure you want to permanently delete the following?\n" +
+            $" >>> {Mathf.Clamp(SaveHelper.CountSaves("auto_save").count - 1, 0, Settings.Instance.MaxAutoSaves)} " +
+            "Auto save(s)\n" +
+            $" >>> {Mathf.Clamp(SaveHelper.CountSaves("quick_save").count - 1, 0, Settings.Instance.MaxQuickSaves)} " +
+            "Quick save(s)";
+        deleteOldConfirmDialog.PopupCenteredMinsize();
+    }
+
     private void OnConfirmDeleteSelected()
     {
         GD.Print("Deleting save(s): ", string.Join(", ", Selected.Select(item => item.SaveName).ToList()));
 
         Selected.ForEach(item => SaveHelper.DeleteSave(item.SaveName));
-        deleteSelectedButton.Disabled = true;
         selected = null;
 
         RefreshList();
     }
 
+    private void OnConfirmDeleteOld()
+    {
+        string autoSaveNames = string.Join(", ", SaveHelper.CleanUpOldSaves("auto_save"));
+        string delimiter = string.Concat(Enumerable.Repeat(", ", Mathf.Clamp(autoSaveNames.Length, 0, 1)));
```

Comment:
What does this do? If you are checking for if anything was added, then just use an if and check if autoSaveNames is length of < 1

Reply:
Yes, it either gives a comma separating auto save names and quick save names, or gives empty string when there are no auto save names. I added this line to reduce the line count since if statement would take more lines. I can change it to an if statement though.

Reply:
I would prefer the if for readablility reason as it took a minute for me to figure out what this code does, whereas conditionally adding a comma is very quick to understand when reading through it the first time.

Reply:
Ok, if it is for readability then it is pretty annoying to read indeed.